<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiplierSettingsDialog</class>
 <widget class="QDialog" name="MultiplierSettingsDialog">
  <property name="windowTitle">
   <string>倍率设置</string>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>16</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>倍率设置</string>
     </property>
     <property name="objectName">
      <string>dialogTitle</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="settingsPanel">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="objectName">
      <string>settingsPanel</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>10</number>
      </property>
      <property name="verticalSpacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelFast">
        <property name="text">
         <string>快速流程倍率</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="fastSpinBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelNormal">
        <property name="text">
         <string>正常流程倍率</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="normalSpinBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelChecked">
        <property name="text">
         <string>勾选超时倍率</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="checkedSpinBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="tipLabel">
     <property name="text">
      <string>范围：0.001–3.000，超出范围将无法保存。</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="objectName">
      <string>dialogSubtleText</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok|QDialogButtonBox::Cancel</set>
     </property>
     <property name="objectName">
      <string>dialogButtons</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
